### DISCUSSION TITLE
Homebrew integration

### DISCUSSION THREAD

**Discussion by rsanjuan87** [oleksis/picta-dl-gui#4] [Ideas]
Ahí está como lo hice funcionar en el homebrew
antes de darle installar al picta-dl hay q hacer un tap al repo, en este caso seria 
`brew tap rsanjuan87/tap`
y luego instalar normal con el brew
`brew install picta-dl`
https://github.com/rsanjuan87/homebrew-tap/blob/main/picta-dl.rb

**Comment by oleksis** [comment]
> Ahí está como lo hice funcionar en el homebrew
> antes de darle installar al picta-dl hay q hacer un tap al repo, en este caso seria
> `brew tap rsanjuan87/tap`

Puedes actualizar el [README.md](https://github.com/oleksis/picta-dl#installation) la sección de Installation para el caso con brew ?

> y luego instalar normal con el brew
> `brew install picta-dl`

Que sería con MacPorts? Algo como:

>> sudo port install picta-dl

**Reply by oleksis** [reply]
Gracias 🙏🏾 por el PR.
